Launcher with autoupdater.py

```diff
@@ -78,21 +39,18 @@ def try_remove_dir(dir):
 		shutil.rmtree(dir)
 		
 def downloadNewestmod():
-	print("test")
+	
 	InstallDir = AppdataPATH
 	
-	launchUrl ="https://api.github.com/repos/OpenGOAL-Unofficial-Mods/OpenGoal-ModLauncher-dev/releases"
-	PARAMS = {'address':"yolo"} 
-	r = json.loads(json.dumps(requests.get(url = launchUrl, params = PARAMS).json()))
+	launchUrl ="https://api.github.com/repos/OpenGOAL-Unofficial-Mods/OpenGoal-ModLauncher-dev/releases"  
+	r = json.loads(json.dumps(requests.get(url = launchUrl, params = {'address':"yolo"}).json()))
```

Comment:
do we even need the params? I think I tried once w/o and it worked the same but never checked it in

Reply:
It didnt work when I tested If I recall correctly
I can test if I remeber tho